src/conditionals.c

```diff
@@ -28,8 +32,12 @@ uint8_t supportsRealKeyboard(void)
 
 uint8_t supportsVirtualKeyboard(void)
 {
-#if defined(__vita__) || defined(__SWITCH__) || defined(_3DS) || defined(__PSL1GHT__)
+#if defined(__vita__) || defined(_3DS) || defined(__PSL1GHT__)
     return 1;
+#elif defined(__SWITCH__)
+    // If app is ran in applet mode, SwKbd will not work
+    AppletType at = appletGetAppletType();
+    return at == AppletType_Application || at == AppletType_SystemApplication ? 1 : 0;
```

Comment:
I think you can just do this?
```suggestion
    return at == AppletType_Application || at == AppletType_SystemApplication;
```

Reply:
My C is kind of rusty, and was being paranoid, you are totally right 👍